mmv1/products/securitycenterv2/FolderMuteConfig.yaml

```diff
@@ -0,0 +1,114 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'FolderMuteConfig'
+base_url: '{{folder}}/muteConfigs'
+self_link: '{{name}}'
+create_url: '{{folder}}/muteConfigs?muteConfigId={{name}}'
+update_verb: :PATCH
+update_mask: true
+description: |
+  Mute Findings is a volume management feature in Security Command Center
+  that lets you manually or programmatically hide irrelevant findings,
+  and create filters to automatically silence existing and future
+  findings based on criteria you specify.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v2/folders.locations.muteConfigs'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_v2_folder_mute_config_basic'
+    primary_resource_id: 'default'
+    vars:
+      mute_config_id: 'my-config'
+    test_env_vars:
+      org_id: :ORG_ID
+# Skipping sweeper since this is a child resource
+skip_sweeper: true
+import_format: ['{{name}}']
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_import: templates/terraform/custom_import/scc_mute_config.go.erb
+parameters:
+  - !ruby/object:Api::Type::String
+    name: muteConfigId
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Unique identifier provided by the client within the parent scope.
+  - !ruby/object:Api::Type::String
+    name: location
+    required: false
+    immutable: true
+    url_param_only: true
+    default_value: global
+    description: |
+      location Id is provided by organization. If not provided, Use global as default.
+  - !ruby/object:Api::Type::String
+    name: folder
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Resource name of the new organization mute configs's parent. Its format is
```

Comment:
It would probably be simpler to add the `folders/` part to the url and remove it from this field. That way the field doesn't need to preserve a forward slash in its value.

Reply:
addressed